pre-docs/inventory_version2.md

```diff
@@ -180,6 +180,93 @@ groups:
             hostname: facts.networking.interfaces.en0.ipaddress
 ```
 
+#### Task
```

Comment:
It might be worth making a bit more noise about this, since users can write their own inventory plugins now. Maybe a level-3 heading of "Writing Inventory Plugins as Tasks" or something?

Reply:
This is not writing your own plugin. You are always using the `task` plugin. I can move it to the top though?